client/lobbies/loading.jsx

```diff
@@ -241,12 +242,13 @@ const GameTypeMapBridge = styled(Display1)`
   color: ${colorTextSecondary};
 `
 
-const MapThumbnail = styled.img`
+const MapThumbnail = styled.div`
   ${shadow1dp};
   width: 256px;
   height: auto;
   border-radius: 2px;
   margin-top: 16px;
+  overflow: hidden;
```

Comment:
I actually added it as a way to show the rounded corners on the map image (since this is now a container that has MapImage as a child). It was either this or styling the map image directly; not sure which way is preferable?

Reply:
Ah, forgot about rounded corners, yeah that's fine 👍